src/SphereXProtectedBase.sol

```diff
@@ -68,7 +68,7 @@ abstract contract SphereXProtectedBase {
      * @param slot where to store the address
      * @param newAddress address to store in given slot
      */
-    function _setAddress(bytes32 slot, address newAddress) private {
+    function _setAddress(bytes32 slot, address newAddress) internal {
```

Comment:
Not sure there's an alternative, but this is a highly sensitive function. Not sure I feel well in exposing it, but I also not sure there's an alternative

Reply:
Maybe reimplement as `private` in derived contracts